meta-xt-qt5/classes/cmake_qt5.bbclass

```diff
@@ -0,0 +1,35 @@
+inherit cmake
```

Comment:
You are adding meta-xt-qt5 layer in the first commit, but looks like it should be done in the commit "[meta-xt-prod-cockpit-rcar-qt5: add layer to support the qt5.](https://github.com/xen-troops/meta-xt-prod-cockpit-rcar/pull/2/commits/efdad20fd8d51b36893f7b0a4b1f36980c7d4b7c)"